README.md

```diff
@@ -100,50 +248,137 @@ This will keep the global `editor.formatOnSave` for other languages you use, and
+You can set the path Terraform logs (TF_LOG_PATH) by configuring this setting:
+
+```json
+"terraform-ls.terraformLogFilePath": "C:/some/folder/path/log-{{.varName}}.log"
+```
+
+Supports variables (e.g. Timestamp, Pid, Ppid) via Go template syntax `{{.VarName}}`
+        
+
+### Telemetry
+
+We use telemetry to send error reports to our team, so we can respond more effectively. You can configure VS Code to send all telemetry, just crash telemetry, just errors or turn it off entirely by [configuring](https://code.visualstudio.com/docs/getstarted/telemetry#_disable-telemetry-reporting) `"telemetry.telemetryLevel"` to your desired value. You can also [monitor what's being sent](https://code.visualstudio.com/docs/getstarted/telemetry#_output-channel-for-telemetry-events) in your logs.
 
 ## Known Issues
 
-* Completion inside incomplete blocks, such as `resource "here` (without the closing quote and braces) is not supported. You can complete the 1st level blocks though and that will automatically trigger subsequent completion for e.g. resource types. See [relevant issue](https://github.com/hashicorp/terraform-ls/issues/57) for more.
-* A number of different folder configurations (specifically when your root module is not a parent to any submodules) are not yet supported. ([info](https://github.com/hashicorp/terraform-ls/issues/32#issuecomment-649707345))
+- If there are validation errors in the open document, intellisense does not provide inline completions. Run `Terraform: validate` and fix validation errors, then reload the document and intellisense will work again. This is being worked on in [terraform-vscode#123](https://github.com/hashicorp/terraform-vscode/issues/123).
+- Completion inside incomplete blocks, such as `resource "here` (without the closing quote and braces) is not supported. You can complete the 1st level blocks though and that will automatically trigger subsequent completion for e.g. resource types. See [terraform-ls#57](https://github.com/hashicorp/terraform-ls/issues/57) for more information.
+- A number of different folder configurations (specifically when your root module is not a parent to any submodules) are not yet supported. More information available in ([terraform-ls#32](https://github.com/hashicorp/terraform-ls/issues/32#issuecomment-649707345))
+
+### Terraform 0.11 compatibility
+
+If you are using a Terraform version prior to 0.12.0, you can install the pre-transfer 1.4.0 version of this extension by following the instructions in the [pin version section](#pin-to-a-specific-version-of-the-extension).
+
+The configuration has changed from 1.4.0 to v2.X. If you are having issues with the Language Server starting, you can reset the configuration to the following:
+
+```json
+"terraform.languageServer": {
+  "external": true,
+  "args": ["serve"]
+}
+```
```

Comment:
I think in the long run we may want to move the 0.11 stuff to a separate file so it doesn't take the expensive estate here in the main Readme.

Reply:
It's been almost 2 years since 2.0.0 was released, I'm fine with moving it now